ckeditor_uploader/views.py

```diff
@@ -110,25 +113,7 @@ def post(self, request, **kwargs):
     def _save_file(request, uploaded_file):
         filename = get_upload_filename(uploaded_file.name, request.user)
 
-        img_name, img_format = os.path.splitext(filename)
-        IMAGE_QUALITY = getattr(settings, "IMAGE_QUALITY", 60)
-
-        if (str(img_format).lower() == "png"):
-
-            img = Image.open(uploaded_file)
-            img = img.resize(img.size, Image.ANTIALIAS)
-            saved_path = storage.save("{}.jpg".format(img_name), uploaded_file)
-            img.save("{}.jpg".format(img_name), quality=IMAGE_QUALITY, optimize=True)
-
```

Comment:
Why do you need to save it as .jpg instead of leaving its original extension? This is causing a problem as I mentioned before.